botx/bot/bot.py

```diff
@@ -145,11 +145,17 @@ def async_execute_raw_bot_command(self, raw_bot_command: Dict[str, Any]) -> None
         self.async_execute_bot_command(bot_command)
 
     def async_execute_bot_command(self, bot_command: BotCommand) -> None:
+        self.ensure_bot_id_exists(bot_command.bot_id)
+
         task = asyncio.create_task(
             self._handler_collector.handle_bot_command(bot_command, self),
         )
         self._tasks.add(task)
 
+    def ensure_bot_id_exists(self, bot_id: UUID) -> None:
```

Comment:
Are you sure this method must be public?

Reply:
This method should be public, but in `BotAccountsStorage`, not in `Bot`.

Reply:
done